locales/en/app.ftl

```diff
@@ -233,6 +233,7 @@ terms-of-service = Terms of Service
 privacy-notice = Privacy Notice
 github = { -brand-github }
 footer-nav-all-breaches = All Breaches
+footer-external-link-how-it-works-label = How It Works
```

Comment:
This still relates to my previous question: this file is localized, the "How It Works" page isn't. Should we have this string in this file? Is it displayed for locales that are not English?